placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesAutocompleteTextView.java

```diff
@@ -145,10 +171,39 @@ public void onItemClick(final AdapterView<?> parent, final View view, final int
                 }
             }
         });
-
+        if(clearEnabled)
+            enableClearButton(true);
         super.setDropDownBackgroundResource(R.drawable.pacv_popup_background_white);
     }
 
+    private void enableClearButton(boolean value){
+        if(!value) {
+            this.setCompoundDrawables(null, null, null, null);
+            return;
+        }
+        if(imgClearButton == null)
```

Comment:
Nit: prefer to use `{` here instead of shorthand. we try to follow google's java style guide

Reply:
Fixed